src/ceph_fuse.cc

```diff
@@ -295,6 +295,8 @@ int main(int argc, const char **argv, const char *envp[]) {
     //cout << "child done" << std::endl;
     return r;
   } else {
+    if (restart_log)
```

Comment:
maybe we'd better use `global_init_prefork()` before the `fork()` and call `global_init_postfork_start()` and `global_init_postfork_finish()` after `fork()`. also, why don't we use the `Preforker` here?